configuration/src/test/java/io/airlift/configuration/TestConfigurationFactory.java

```diff
@@ -277,6 +282,46 @@ public void testFailedCoercion()
                 ".*Invalid value 'abc %s xyz' for type int \\(property 'int-value'\\).*BeanValidationClass.*");
     }
 
+    @Test
+    public void testSuccessfulFileExistsValidation()
```

Comment:
This test class is for testing that `ConfigurationFactory` works and that it correctly uses bean validation for config classes. It's not for testing that specific validators work -- they should be tested directly using the validation framework.

Look at the tests for https://github.com/airlift/units to see how to write a validator. You can import the `test-jar` for `units` to reuse `ConstraintValidatorAssert`.

Reply:
I can't use that. ConstraintValidatorAssert is package private